### ISSUE TITLE
ASCII data file with attached label (.DAT) does not load any objects

### ISSUE THREAD

**Issue by msbentley** [MillionConcepts/pdr#12]
Trying to load this VEX/MAG product:

ftp://psa.sciops.esa.int/pub/mirror/VENUS-EXPRESS/MAG/VEX-V-Y-MAG-2-EXT4-V1.0/DATA/ORB201411_D001/BIO_20141125_DOY329_D001_V1.TAB

everything looks OK but no data objects are shown:

```
In [13]: data = read('MAG_20141125_DOY329_D001_V1.TAB')

In [14]: data.keys()
Out[14]: []
```

**Comment by m-stclair** [comment]
as of version 1.2.1, pvl refuses to read a label from this file because it has several improperly formatted NOTE parameters, which is most likely the underlying cause of this bug

**Comment by michaelaye** [comment]
i'm sure Ross would love to look into this!

**Comment by m-stclair** [comment]
in this case I think it is the label's 'fault' in the sense that it is not strictly-valid PVL...I think that we should have a discussion with Ross at some point about exactly what he sees as the intended parsing scope of `pvl`-- there's a _lot_ of technically-invalid PVL in PDS legacy archives, a lot of it much worse than this, and I don't know what classes of bad formatting he does and doesn't  want to support.

**Comment by msbentley** [comment]
Yes, that would be interesting - perhaps to have a `strict` parameter, or similar, where one could allow some of the more "funky" labels ;-)

**Comment by michaelaye** [comment]
`pvl` already allows 3 or 4 "dialects" so there's definitely the intent and appreciation that not everything is the same `pvl` standard. I think it's worth creating an issue at `pvl`.

**Comment by msbentley** [comment]
I had a look at the label in question and indeed there is a UTF-8 character in the NOTE field that, when removed, allows `pvl` to correctly read the data...
